test/test_sox_compatibility.py

```diff
@@ -1,510 +1,304 @@
-
-        waveform_gain = F.gain(waveform, 3)
-        self.assertTrue(waveform_gain.abs().max().item(), 1.)
-
-        E = torchaudio.sox_effects.SoxEffectsChain()
-        E.set_input_file(test_filepath)
-        E.append_effect_to_chain("gain", [3])
-        sox_gain_waveform = E.sox_build_flow_effects()[0]
-
-        self.assertEqual(waveform_gain, sox_gain_waveform, atol=1e-04, rtol=1e-5)
+        path = get_asset_path('steam-train-whistle-daniel_simon.wav')
+        data, _ = load_wav(path)
+        result = F.gain(data, 3)
+        self.assert_sox_effect(result, path, ['gain', 3])
 
     def test_dither(self):
-        test_filepath = common_utils.get_asset_path('steam-train-whistle-daniel_simon.wav')
-        waveform, _ = common_utils.load_wav(test_filepath)
-
-        waveform_dithered = F.dither(waveform)
-        waveform_dithered_noiseshaped = F.dither(waveform, noise_shaping=True)
+        path = get_asset_path('steam-train-whistle-daniel_simon.wav')
+        data, _ = load_wav(path)
+        result = F.dither(data)
+        self.assert_sox_effect(result, path, ['dither'])
 
-        E = torchaudio.sox_effects.SoxEffectsChain()
-        E.set_input_file(test_filepath)
-        E.append_effect_to_chain("dither", [])
-        sox_dither_waveform = E.sox_build_flow_effects()[0]
-
-        self.assertEqual(waveform_dithered, sox_dither_waveform, atol=1e-04, rtol=1e-5)
-        E.clear_chain()
-
-        E.append_effect_to_chain("dither", ["-s"])
-        sox_dither_waveform_ns = E.sox_build_flow_effects()[0]
-
-        self.assertEqual(waveform_dithered_noiseshaped, sox_dither_waveform_ns, atol=1e-02, rtol=1e-5)
-
-    def test_vctk_transform_pipeline(self):
-        test_filepath_vctk = common_utils.get_asset_path('VCTK-Corpus', 'wav48', 'p224', 'p224_002.wav')
-        wf_vctk, sr_vctk = common_utils.load_wav(test_filepath_vctk)
-
-        # rate
-        sample = T.Resample(sr_vctk, 16000, resampling_method='sinc_interpolation')
-        wf_vctk = sample(wf_vctk)
-        # dither
-        wf_vctk = F.dither(wf_vctk, noise_shaping=True)
-
-        E = torchaudio.sox_effects.SoxEffectsChain()
-        E.set_input_file(test_filepath_vctk)
-        E.append_effect_to_chain("gain", ["-h"])
-        E.append_effect_to_chain("channels", [1])
-        E.append_effect_to_chain("rate", [16000])
-        E.append_effect_to_chain("gain", ["-rh"])
-        E.append_effect_to_chain("dither", ["-s"])
-        wf_vctk_sox = E.sox_build_flow_effects()[0]
-
-        self.assertEqual(wf_vctk, wf_vctk_sox, rtol=1e-03, atol=1e-03)
+    def test_dither_noise(self):
+        path = get_asset_path('steam-train-whistle-daniel_simon.wav')
+        data, _ = load_wav(path)
+        result = F.dither(data, noise_shaping=True)
+        self.assert_sox_effect(result, path, ['dither', '-s'], atol=1.5e-4)
 
     def test_lowpass(self):
-        """
-        Test biquad lowpass filter, compare to SoX implementation
-        """
```

Comment:
Why is the description removed?

Reply:
Because all the tests in the class have the same description. It's redundant and does not carry new useful information.